TYPO3.Neos/Resources/Private/Styles/Foundation/_labels-badges.scss

```diff
@@ -7,7 +7,7 @@
 .neos-label,
 .neos-badge {
   display: inline-block;
-  padding: 2px 4px;
+  padding: 4px 6px;
```

Comment:
yeah makes sense, but don't use BEM modifiers.. stick with how it's done in bootstrap, like `.neos-label-large`